rules/falco-sandbox_rules.yaml

```diff
@@ -1692,4 +1692,25 @@
   priority: WARNING
   tags: [maturity_sandbox, container, filesystem, mitre_initial_access, T1611]
 
-  
+- list: known_decode_payload_containers
+  items: []
+
+- macro: base64_decoding
+  condition: (proc.cmdline contains "base64" and (proc.cmdline contains "--decode" or proc.cmdline contains "-d"))
+
+- rule: Decoding Payload in Container
+  desc: > 
+    Detect any use of {base64} decoding in a container. Legitimate applications may decode encoded payloads. The template list 
+    known_decode_payload_containers can be used for simple tuning and customization, or you can adopt custom, more refined tuning. Less 
+    sophisticated adversaries may {base64}-decode their payloads not only to obfuscate them, but also to ensure that the payload remains 
+    intact when the application processes it. Note that injecting commands into an application's input often results in the application 
+    processing passed strings like "sh -c". In these cases, you may be lucky and the encoded blob will also be logged. Otherwise, all you 
+    will see is the {base64} decoding command, as the encoded blob was already interpreted by the shell. 
+  condition: >
+    spawned_process 
+    and container 
+    and base64_decoding 
```

Comment:
In general, rule is composed in a way that we can add more types of decoding as we see fit, for now only base64 decoding.